onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -269,10 +269,33 @@ def aten_angle(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_any(self: TensorType) -> TensorType:
+@torch_op("aten::any", trace_only=True)
```

Comment:
Why do we need trace_only here? Looks like all of operations are safe.

Reply:
Could you say more on what optional argument parsing is? I would also add a comment on why trace_only was needed in code

Reply:
if not use trace_only, it will throw exception:

---
E   TypeError: Required input 'dim: Attribute[None]' was not provide
---

Reply:
I think this is the same as the issue we discussed in today's meeting. `if (OptionalHasElement(x)) then use(x)` is an issue. We need to find a solution to this. One possibility is to extend OptionalGetElement, but wonder if that's enough for all cases.